modules/rosa-deleting-cluster.adoc

```diff
@@ -125,8 +125,13 @@ $ rosa delete oidc-provider -c <cluster_id> --mode auto <1>
 ====
 You can use the `-y` option to automatically answer yes to the prompts.
 ====
-
-. Delete the cluster-specific Operator IAM roles:
++
+. Optional. Delete the cluster-specific Operator IAM roles:
++
+[IMPORTANT]
+====
+The account-wide IAM roles can be used by other ROSA clusters in the same AWS account. You must only remove the roles if they are not required by other clusters.
```

Comment:
s/You must only/Only   Maybe? It seems odd to have a _You must_ in an optional step. I could be wrong.

Reply:
If I really wanted to be picky, I would say _Rnly remove the roles only if they are not required by other clusters._

Reply:
I like it! Thanks! :)